src/components/molecules/SettingAgents.tsx

```
@@ -0,0 +1,98 @@
+                        </div>
+                    </div>
+                    {agentName === openDescription && <div className="mt-2 ml-8">
+                        {agents.length === 0 ? (
+                            <div className="py-3 px-6 mb-2 border border-neutral-300 rounded-md">
+                                <p>利用可能なエージェント情報がありません。</p>
+                            </div>
+                        ) : (<>
+                            <div className="px-6 py-3 mb-4 border border-neutral-300 rounded-md">
+                                {agents.filter(agent => agent.name === agentName).map((t) => (
+                                    <div className="py-2 px-6 mb-2 flex gap-4 flex-col">
+                                        <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)